src/analysis/retail/hunter/survival/modules/apl/AplCheck.tsx

```diff
@@ -0,0 +1,196 @@
+import SPELLS from 'common/SPELLS';
+import TALENTS from 'common/TALENTS/hunter';
+import RESOURCE_TYPES from 'game/RESOURCE_TYPES';
+import SpellLink from 'interface/SpellLink';
+import { suggestion } from 'parser/core/Analyzer';
+import { AnyEvent } from 'parser/core/Events';
+import aplCheck, { build, CheckResult, PlayerInfo, tenseAlt } from 'parser/shared/metrics/apl';
+import annotateTimeline from 'parser/shared/metrics/apl/annotate';
+import { AplRuleProps } from 'parser/shared/metrics/apl/ChecklistRule';
+import * as cnd from 'parser/shared/metrics/apl/conditions';
+
+const withTip = cnd.buffPresent(SPELLS.TIP_OF_THE_SPEAR_CAST);
+const hasFocus = (min: number) => cnd.hasResource(RESOURCE_TYPES.FOCUS, { atLeast: min });
+export const apl = build([
+  {
+    spell: TALENTS.KILL_COMMAND_SURVIVAL_TALENT,
+    condition: cnd.describe(
+      cnd.and(
+        cnd.hasTalent(TALENTS.RELENTLESS_PRIMAL_FEROCITY_TALENT),
+        cnd.buffMissing(SPELLS.TIP_OF_THE_SPEAR_CAST),
+        cnd.buffPresent(SPELLS.COORDINATED_ASSAULT_BUFF),
+      ),
+      (tense) => <>you have 0 tip stacks & Relentless Primal Ferocity is active</>,
```

Comment:
```suggestion
      (tense) => <>you {tenseAlt(tense, 'have', 'had')} 0 tip stacks & Relentless Primal Ferocity is active</>,
```

use `tenseAlt` to adjust the tense of the description so that it is correct when shown in past-tense contexts (usually describing rotational errors, ex: "you should have cast this because you had 0 tip stacks" instead of "you should have cast this because you have 0 tip stacks")